datmo/test/test_task.py

```diff
@@ -0,0 +1,57 @@
+"""
+Tests for snapshot module
+"""
+import os
+import shutil
+import tempfile
+
+from datmo.task import run
```

Comment:
Yea we could do the same for snapshot. we should consult with @nmwalsh about the interface. 

Basically we can either make it just `datmo.task.run` or `datmo.snapshot.create` or we can make it like static methods of a class `datmo.task.Task.run` and `datmo.snapshot.Snapshot.create`, but the issue with that is that this will be confused with the Task and Snapshot entities which each have an id.

Reply:
Also we should consider this scenario for remote runs
```python
from datmo.task import Task
from datmo.cluster import Cluster

cluster = Cluster.create()
task1 = Task(command="echo HELLO")

cluster.exec(task1)
# OR
task1.run(cluster)

```

Reply:
Our initial interface was this way
```
import datmo
datmo.task.run()
```
Having it as you proposed would definitely decrease the time for TaskController instantiation